CrewChiefV4/Audio/PlaybackModerator.cs

```diff
@@ -166,7 +166,11 @@ public static bool ShouldPlaySound(SingleSound sound, Boolean canInterrupt)
                 PlaybackModerator.Trace(string.Format("Sound {0} rejected because we're in the middle of a voice command", sound.fullPath));
                 return false;
             }
-
+            if (CrewChief.currentGameState != null && CrewChief.currentGameState.IsInHardPartOfTrack && canInterrupt)
```

Comment:
I agree with Jim here, this blocking should be optional, because currently it is producing weird results sometime.